libbeat/docs/shared-template-load.asciidoc

```diff
@@ -107,6 +110,7 @@ ifeval::["{beatname_lc}"!="auditbeat"]
 docker run {dockerimage} setup --template -E output.logstash.enabled=false -E 'output.elasticsearch.hosts=["localhost:9200"]'
 ----------------------------------------------------------------------
 
+
```

Comment:
Adding double line breaks after examples because it fixes rendering issues when these files are viewed on GitHub.